http.go

```diff
@@ -117,6 +130,59 @@ func (r *httpRequest) encodeBody() (io.Reader, error) {
 	}
 }
 
+func (r httpRequest) isMultipartFormDataMediaType() bool {
```

Comment:
need pointer?

```suggestion
func (r *httpRequest) isMultipartFormDataMediaType() bool {
```

Reply:
no need pointer because this is not mutable method🙋 If it is convention, I follow you👍

Reply:
> no need pointer because this is not mutable method

You are certainly right.

> If it is convention, I follow you

It is not convention. I had made it a pointer unconditionally, so I learned a lot. Thank you!